src/java.base/share/classes/sun/security/util/DerValue.java

```diff
@@ -610,6 +626,17 @@ else if (tag == tag_GeneralString)
             return null;
     }
 
+    public DerValue[] getSubs(byte type) throws IOException {
```

Comment:
Add comments? It's not very clear what getSubs(...) does just by the method name... It seems to parse and return all DerValue objs from the underlying data bytes?